jwql/jwql_monitors/generate_proposal_thumbnails.py

```diff
@@ -34,14 +34,15 @@
 from jwql.utils.utils import get_config
 from jwql.utils.monitor_utils import initialize_instrument_monitor, update_monitor_table
 
+SETTINGS = get_config()
```

Comment:
Yes. The idea behind this is that I found that running get_config() actually takes quite a while. In one instance there was a call to get_config() inside of a loop over a list of files, and this was adding a significant amount of run time to the script. So my thought was to call get_config() once and have the results as a  global so that all of the functions could then simply access the resulting dictionary.